### ISSUE TITLE
Add ``minimal_pull`` boolean kwarg to ``make_md_workflow`` for higher throughput

### ISSUE THREAD

**Issue by dotsdl** [Becksteinlab/mdworks#4]
At the cost of making the workflow a bit more complex, it would be a performance boost to follow up the MD Firework with a `MinimalPull` FireTask that pulls back only the files necessary to begin the next run (which, unless I'm mistaken, should be the files specified in the `files` kwarg). These files would need to be sufficient for the `ContinueTask` of the MD engine being used to make its decision.

Basically, `MinimalPull` has two children: `Continue` and `Pull`. The advantage is that the next job, if necessary, can be ready to go while the data from the last one is still being pulled back to the archive. We don't have to wait for a full transfer to keep going.

We could make workflow configuration optional, since it introduces more complexity and may not be suitable for all MD engines. Something like `minimal_pull=True` could be set to use it, defaulting to `False`.

**Comment by orbeckst** [comment]
Before implementing this I would try to find out if this is actually a problem at the moment.

**Comment by dotsdl** [comment]
Yeah it's not a priority at the moment. With enough simulations waiting in the wings copyback is not a bottleneck. I jotted this here so I wouldn't forget the idea. I'll remove it from the first release milestone.
